bin/loklakinit.py

```diff
@@ -9,151 +10,236 @@
+  app_json["applicationSubCategory"] = ""
 
+  # Value must be a string
+  app_json["operatingSystem"] = "http://loklak.org"
 
-  app_alternative_headline = raw_input("alternative headline (" + app_json["alternativeHeadline"] + ") : ")
-  if app_alternative_headline :
-    app_json["alternativeHeadline"] = app_alternative_headline
+  # Value must be a string
+  app_json["promoImage"] = "promo.png"
 
-  app_category = raw_input("application category (" + app_json["applicationCategory"] + ") : ")
-  if app_category :
-    app_json["applicationCategory"] = app_category
+  # Value must be a string
+  app_json["appImages"] = ""
 
-  app_sub_category = raw_input("sub category : ")
-  if app_sub_category :
-    app_json["applicationSubCategory"] = app_sub_category
+  # Value must be a string
+  app_json["oneLineDescription"] = ""
 
-  app_os = raw_input("application operating system (" + app_json["operatingSystem"] + ") : ")
-  if app_os :
-    app_json["operatingSystem"] = app_os
+  # Value must be a string
+  app_json["getStarted"] = "getStarted.md"
 
-  app_promo_image = raw_input("promo image (" + app_json["promoImage"] + ") : ")
-  if app_promo_image :
-    app_json["promoImage"] = app_promo_image
+  # Value must be a string
+  app_json["appUse"] = "appUse.md"
 
-  app_images = raw_input("app preview images (values can be separted by comma) : ")
-  if app_images :
-    app_json["appImages"] = app_images.split(",")
+  # Value must be a string
+  app_json["others"] = "others.md"
 
-  app_description = raw_input("one line description : ")
-  if app_description :
-    app_json["oneLineDescription"] = app_description
+  # Value must be a string
+  app_json["appSource"] = ""
 
-  author["name"] = raw_input("author name : ")
+  ''' Value must be a list containing dictionaries of form -
+  {"name": name_of_contributor,
+   "url": url_of_contributor
+  }
+  '''
+  app_json["contributors"] = ""
 
-  author_type = raw_input("author type (" + author["@type"] + ") : ")
-  if author_type :
-    author["@type"] = author_type
+  # Value must be a list containing strings
+  app_json["techStack"] = ""
 
-  author["email"] = raw_input("email : ")
+  ''' Value must be a dictionary of form -
+  {"name": license_name,
+   "url": license_url}
+  '''
+  app_json["license"] = {"name": "LGPL 2.1", "url": "https://www.gnu.org/licenses/old-licenses/lgpl-2.1"}
 
-  author["url"] = raw_input("url : ")
-  author["sameAs"] = author["url"]
+  # Value must be a string
+  app_json["version"] = "1.0"
 
-  author["sameAs"] = raw_input("same as : ")
+  #Value must be string
+  app_json["updated"] = ""
 
+  author = collections.OrderedDict()
+
+  # Value must be a string
+  author["@type"] = "Person"
+
+  # Value must be a string
+  author["name"] = ""
+
+  # Value must be a string
+  author["email"] = ""
+
+  # Value must be a string
+  author["url"] = ""
+
+  # Value must be a string
+  author["sameAs"] = ""
+
+  # Value must be author dictionary
   app_json["author"] = author
 
```

Comment:
Is there a reason the `app_json["license"]` is hardcoded and not taken as an input from the user?